metadata-ingestion/src/datahub/ingestion/api/workunit.py

```diff
@@ -46,9 +59,11 @@ def __init__(
         mcp: Optional[MetadataChangeProposalWrapper] = None,
         mcp_raw: Optional[MetadataChangeProposal] = None,
         treat_errors_as_warnings: bool = False,
+        is_primary_source: bool = True,
     ):
         super().__init__(id)
         self.treat_errors_as_warnings = treat_errors_as_warnings
+        self.is_primary_source = is_primary_source
```

Comment:
why are we even emitting stuff for any entities where we are not the primary source for it>?

Reply:
siblings requires that the snowflake node has an upstream pointing at the dbt model

that means we need to emit that sibling from here